src/test/java/api/loans/EndExpiredPatronActionSessionTests.java

```diff
@@ -274,4 +279,63 @@ public void expiredSessionWithNonExistentUserShouldBeEnded() {
       .atMost(1, TimeUnit.SECONDS)
       .until(patronSessionRecordsClient::getAll, empty());
   }
+
+  @Test
+  public void shouldNotFailWithUriTooLargeErrorDuringEndingExpiredCheckOutSessions() {
+
+    createOneHundredPatronActionCheckOutSessions("Check-out");
+    List<JsonObject> sessions = patronSessionRecordsClient.getAll();
+    assertThat(sessions, Matchers.hasSize(100));
+
+    expiredSessionProcessingClient.runRequestExpiredSessionsProcessing(204);
+
+    Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(patronSessionRecordsClient::getAll, empty());
+  }
+
+  @Test
+  public void shouldNotFailWithUriTooLargeErrorDuringEndingExpiredCheckInSessions() {
+
+    createOneHundredPatronActionCheckOutSessions("Check-in");
+    List<JsonObject> sessions = patronSessionRecordsClient.getAll();
+    assertThat(sessions, Matchers.hasSize(100));
+
+    expiredSessionProcessingClient.runRequestExpiredSessionsProcessing(204);
+
+    Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(patronSessionRecordsClient::getAll, empty());
+  }
+
+  @Test
+  public void sessionsWithNotSpecifiedActionTypeShouldBeEnded() {
+
+    createOneHundredPatronActionCheckOutSessions("");
+    List<JsonObject> sessions = patronSessionRecordsClient.getAll();
```

Comment:
Please extract this code into a private method

Reply:
Done